scripts/dash-docking-exploration.py

```diff
@@ -0,0 +1,227 @@
+import pandas as pd
```

Comment:
We want a docstring here telling potential users and maintainers what this script does, with inputs and outputs. In a few words.